src/librustdoc/html/static/rustdoc.css

```diff
@@ -462,15 +467,15 @@ nav.sub {
 	font-weight: normal;
 }
 
-h3.impl > .out-of-band {
+div[data-level="1"].impl > .out-of-band {
 	font-size: 21px;
 }
 
-h4.method > .out-of-band {
+div[data-level="2"].method > .out-of-band {
 	font-size: 19px;
 }
```

Comment:
As far as I can tell, `.out-of-band` is only used for the top-right `[src]` link of the whole page, which is not nested under any h3 or h4. I think we can delete these rules.